src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -294,6 +296,8 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
                 return !is_type<ov::op::v0::MatMul>(next_node);
             });
 
+        manager.register_pass<ov::intel_gpu::GroupNormComposition>();
```

Comment:
GroupNormDecomposition is being disabled, but still need Composition too?

Reply:
Current OV stable diffusion models does not have `group_normalization` layers. They have decomposed group_normalization layers instead. So we need a composition pass to apply `group_normalization` in the current OV SD models.